internal/k8s/locator.go

```diff
@@ -121,6 +125,10 @@ func (l *JSONPathImageLocator) Inject(e K8sEntity, selector container.RefSelecto
 		if selector.Matches(ref) {
 			val.Set(reflect.ValueOf(container.FamiliarString(injectRef)))
 			modified = true
+
+			if pullPolicyVal, ok := val.Sibling(imagePullPolicyKey); ok {
```

Comment:
can you also check pullPolicyVal.CanSet()? always get nervous about Value.Set panicking

Reply:
ha, did you see the CI failure?

Reply:
lol, nope i did not! :star2: